### PR TITLE
feat: add Agora Flat as user

### PR DESCRIPTION
**Pull request by BlackHole1**
This is open source project. 
See: https://github.com/netless-io/flat

### CODE REVIEW

File: users.json [hunk 1]
```diff
@@ -281,5 +281,12 @@
     "infoLink": "https://element-plus.org/",
     "pinned": false,
     "width": 250
+  },
+  {
+    "caption": "Agora Flat",
+    "image": "/img/users/agora-flat.svg",
+    "infoLink": "https://flat.whiteboard.agora.io/en/",
+    "pinned": true,
```
Comment (juanpicado) [inline users.json:289]: Usually this is false, could you update?

Comment (BlackHole1) [reply users.json:289]: This is the expected modification 👀. From my personal point of view, I would expect the use of `pinned: true` here.
If you don't feel too good about it, please let me know.

Comment (juanpicado) [reply users.json:289]: I am fine with it, Zoltan has here the last word.

Comment (BlackHole1) [reply users.json:289]: @zkochan

Comment (zkochan) [reply users.json:289]: We usually pin only the most popular projects to the homepage.
Is Agora Flat well known?

Comment (BlackHole1) [reply users.json:289]: Okey 👌

**Pull request merged**